test/js/pageobjects/Welcome.js

```diff
@@ -34,6 +36,14 @@ class Welcome extends Base {
   async verifyFooter() {
     this.footer.isDisplayed()
   }
+
+  async clickOnButtonToOpenModalView() {
```

Comment:
for consistency with the selector's name should be named `clickOnOpenModalButton`

Reply:
Done in: 6bba9d1